app/uk/gov/hmrc/tariffclassificationfrontend/models/Application.scala

```diff
@@ -24,8 +24,24 @@ sealed trait Application {
   def asBTI(): BTIApplication = {
     this.asInstanceOf[BTIApplication]
   }
+
+  def asLiabilityOrder(): LiabilityOrder = {
+    this.asInstanceOf[LiabilityOrder]
+  }
+
+  def isBTI(): Boolean = {
```

Comment:
Just a thought. Was the type created to solve this type of problems?

Reply:
For some reason the `type` field is not directly accessible in the templates :s It needs formatting anyway so its useful to have the `getType` method so we're not showing stuff like "LIABILITY_ORDER" in the UI

Reply:
Theres also going to be some logic around what type of application it is (e.g. showing them in different views) so I think its quite nice to have an `isBTI` `isLiability` method rather than doing `instanceOf` or `type == "BTI"` everywhere.  I'm open to alternatives

Reply:
Also, to be honest I think the type field is for deserialisation, it shouldn't even be in the model really as we can use the inheritance structure to work it out